### PR TITLE
Fix MM57562

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Fix MM-57562

#### Ticket Link
Fix https://mattermost.atlassian.net/browse/MM-57562

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: app/components/math_view/index.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+const MathView = (props: Props) => {
+    const id = useId();
+    const config = useMemo(() => ({id}), []);
+
+    return (
+        <RNMathView
+            {...props}
+            config={config}
```
Comment (enahum) [inline app/components/math_view/index.tsx:25]: How is this fixing the issue?

**Review by hmhealey — APPROVED** [review]

**Review by enahum — APPROVED** [review]

**Comment by larkox** [comment]
@yasserfaraazkhan Here is only needed some smoke testing around latex to make sure nothing has broken.

Both inline (`$some latext$`) and codeblock (triple back tick with language name `latex`).

**Review by yasserfaraazkhan — APPROVED** [review]
in line and code block ✅ 
image


was exploring different multiple line latex code and saw this showing in mobile.
for below example. Just sharing as an observation.

```latex
\documentclass{article}
\usepackage{amsmath}

\begin{document}

Trigonometric equation:
\[ \sin^2(x) + \cos^2(x) = 1 \]

\end{document}
```


Screenshot 2024-05-05 at 11 42 01 PM

**Comment by larkox** [comment]
Thank you @yasserfaraazkhan . Not sure if we want to document that as "known limitation", but looks good to me.

**Pull request merged**